src/scripts/content/anydo.js

```diff
@@ -28,13 +28,13 @@ togglbutton.render(
   '.TaskListRow[draggable=true]:not(.toggl)',
   { observe: true },
   elem => {
-    const descriptionElem = $('input', elem);
+    const descriptionElem = elem.querySelector('.TaskItem__label__text__title').textContent;
```

Comment:
We should change this into a function that returns the description string, like we do in more modern integrations (usually called something like `descriptionSelector` if you need a reference)

This will mean that we'll always grab the latest description, even after the user might have changed it in the `any.do` UI.

Reply:
Thanks! I've updated all of them to work this way :+1: